src/main/ngapp/accio/src/app/editor/set-operator.spec.ts

```diff
@@ -1,25 +1,29 @@
 import { SetOperator } from './set-operator';
 
 describe('SetOperator', () => {
- // TODO
- const setA = new Set([1, 2, 3, 4]);
- const setB = new Set([3, 4, 5, 6]);
+  // TODO
```

Comment:
@charmychancey what is this TODO for?

Reply:
I forgot to remove it, you can go ahead and do so